examples/hello_world/BUILD

```diff
@@ -0,0 +1,47 @@
+cc_library(
+    name = "vsomeip_hello_world_service",
+    hdrs = ["hello_world_service.hpp"],
+    include_prefix = ".",
+)
+
+cc_library(
+    name = "vsomeip_hello_world_client",
+    hdrs = ["hello_world_client.hpp"],
+    include_prefix = ".",
+)
+
+cc_binary(
+    name = "hello_world_service",
+    srcs = [
+        "hello_world_service_main.cpp",
+        "//:libvsomeip3.so"
+    ],
+    deps = [
+        ":vsomeip_hello_world_service",
+        "//interface",
+    ],
+    data = [
+        "//:vsomeip3-plugins"
+    ],
+    linkopts = [
+        "-Wl,-rpath=\\$$ORIGIN/../..,--disable-new-dtags",
```

Comment:
nit: it'd be great if we could transform this o the `pacakge_name` of `//:libvsomeip3.so`